src/SingleView/SingleView.js

```diff
@@ -5,7 +5,22 @@ import './SingleView.css'
 export class SingleView extends Component {
   constructor(props) {
     super(props)
-    this.state = {}
+    this.state = { currentMovie: { id: props.currentMovieID.id } }
+  }
+
+  fetchData = (path) => {
+    return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/${path}`).then((res) => {
+      return res.json()
```

Comment:
I know you trust the API will give you a valid response, but it's a good practice to first double-check that the response was successful. Check out `response.ok` to handle some of this logic.